forward_engineering/helpers/azureCLIScriptHelpers/buildAzureCLIScript.js

```diff
@@ -0,0 +1,205 @@
+const { wrapInSingleQuotes, getEscapeFunction } = require('./escapeShellSpecialCharacters');
+const applyToInstanceHelper = require('../../applyToInstance/applyToInstanceHelper');
+const { getUniqueKeyPolicyScript } = require('../getUniqueKeyPolicyScript');
+const { getCliParamsDelimiter } = require('./getCliParamsDelimiter');
+const getIndexPolicyScript = require('../getIndexPolicyScript');
+const getPartitionKey = require('../getPartitionKey');
+const {
+	CLI,
+	DATABASE,
+	CREATE,
+	CONTAINER,
+	STORED_PROCEDURE,
+	TRIGGER,
+	USER_DEFINED_FUNCTION,
+} = require('./azureCLIConstants');
+
+const buildAzureCLIScript =
+	_ =>
+	({ modelData, containerData, shellName }) => {
+		const cliParamsDelimiter = getCliParamsDelimiter(shellName);
+		const escapeAndWrapInQuotes = string => wrapInSingleQuotes(getEscapeFunction(shellName)(string));
```

Comment:
# Remark 2

After completing Remark 1, update invokation to be "escapeShellCommandBeforeWrappingInQuotes(shellName, string)"